cpp/models/abm/parameters.h

```diff
@@ -245,7 +245,7 @@ struct AerosolTransmissionRates {
     using Type = CustomIndexArray<ScalarType, VirusVariant>;
     static Type get_default(AgeGroup /*size*/)
     {
-        return Type({VirusVariant::Count}, 1.0); // amount of infections per m^3 per day
+        return Type({VirusVariant::Count}, 1.0);
```

Comment:
What was the reason to remove this comment? It would be useful to have the description somewhere, it can also be in `@brief`.

Reply:
i think it was not that descriptive... how many are infected are heavily dependent on amount of people, so it is rather a percentage?

Reply:
That's a good explanation, and i thought that was what it meant, I shouldn't have deleted it, but as a newcomer the comment maybe isn't exhaustive enough, perhaps we can add a comment in your issue? I will resolve this disc and merge the issue